### DISCUSSION TITLE
consider disabling VSCode notification when biome is not found in dependencies

### DISCUSSION THREAD

**Discussion by josefaidt** [biomejs/biome-vscode#233] [General]
Hey folks :wave: each time I open a new VSCode window I am presented with this notification where biome is not found in my dependencies. This appears when the extension starts, without a package.json, etc.. I can opt to disable all warnings and info notifications from the biome extension, but I'd prefer not to in case there are other, helpful notifications.

![CleanShot 2024-06-07 at 08 11 48@2x](https://github.com/biomejs/biome/assets/5033303/749ea624-4169-4754-a4ec-b29bbb6041b6)

**Comment by brunobely** [comment]
Recently this one pops up every so often even without opening a new window - I'm definitely going to be missing actual helpful notifications as I can't keep closing this every 30mins/1h/however long it takes to reappear:
<img width="345" alt="image" src="https://github.com/user-attachments/assets/27de77e1-0efb-4a76-9ad8-85a886b43d27" />

**Reply by nhedger** [reply]
Did you try clicking the "do not show again" or installing Biome globally ?

This popup appear because most likely because you opened your VS Code settings or an Untitled file and the extension nees a global Biome installation to provide formatting and linting to those files.

**Comment by ematipico** [comment]
If I'm not mistaken, there's already a setting for that. It should be `biome.enabled`. Can you try that?
